.editorconfig

```diff
@@ -0,0 +1,10 @@
+[*]
+charset = utf-8
+end_of_line = crlf
```

Comment:
I guess this is wrong, isnt it? I'd prefer the unix linefeed. If some files are not yet in that format, that might be true, as they are very old.